openwisp_users/tests/test_admin.py

```diff
@@ -169,6 +172,36 @@ def test_admin_change_user_email_empty(self):
         self.assertEqual(len(mail.outbox), 0)
         self.assertContains(response, 'errors field-email')
 
+    def test_admin_change_user_page_get_invalid_UUID(self):
+        admin = self._create_admin()
+        self.client.force_login(admin)
+        with self.subTest('Test for wrong identifier'):
+            response = self.client.get(
+                reverse(f'admin:{self.app_label}_user_change', args=['WRONG']),
+                follow=True,
+            )
+            # TODO: Remove this when we drop support for django<3.0
+            if django.VERSION < (3, 1):
+                # check for version < 3.1
+                content = 'User with ID "WRONG" doesn\'t exist. Perhaps it was deleted?'
+            else:
+                content = 'User with ID “WRONG” doesn’t exist. Perhaps it was deleted?'
+            response.content = html.unescape(response.content.decode())
+            self.assertContains(response, content, status_code=200)
+        with self.subTest('Test for non-existing user'):
+            id = uuid.uuid4()
+            response = self.client.get(
+                reverse(f'admin:{self.app_label}_user_change', args=[id],), follow=True
+            )
+            # TODO: Remove this when we drop support for django<3.0
+            if django.VERSION < (3, 1):
+                # check for version < 3.1
+                content = f'User with ID "{id}" doesn\'t exist. Perhaps it was deleted?'
+            else:
+                content = f'User with ID “{id}” doesn’t exist. Perhaps it was deleted?'
```

Comment:
I would have avoided the version switch and checked only for part of the string, however, since it's already done and the check works, we can leave it, thanks.

Reply:
Okay!

Reply:
Do you mean something like this? 
```py
content = '<li class="warning">User with ID '
```